srcpkgs/autorestic/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'autorestic'
+pkgname=autorestic
+version=1.7.7
+revision=1
+build_style=go
+go_import_path="github.com/cupcakearmy/autorestic"
+depends="restic"
```

Comment:
I assume this should be included, despite the binary running without restic, albeit useless, since it can't really do anything useful without it.